content/options.html

```diff
@@ -22,7 +22,45 @@
 
   <label for="github-username">Github Username</label>
   <input type="text" id="github-username" data-type="ghuser">
+
+  <h1>Working Hours</h1>
+  <br>
+
+  <label for="working-hours-checkbox">Only show review count during working hours</label>
+  <input type="checkbox" id="working-hours-checkbox">
+
+  <fieldset id="working-hours" disabled>
```

Comment:
I completely forgot that fieldsets could be mass enabled / disabled like this! Nice!